firefox-ios/Client/metrics.yaml

```diff
@@ -753,6 +753,163 @@ usage:
     send_in_pings:
       - usage-reporting
 
+  duration:
+    type: timespan
+    description: |
+      The duration of the last foreground session.
+    time_unit: second
+    send_in_pings:
+      - usage-reporting
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/23785
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/23950
```

Comment:
Please link to the original Glean duration for the data-review and bugs also, as that is the review that covers this collection. You can find these links in the Glean dictionary for this and other metrics which have been duplicated for the purpose of the usage-reporting ping.

Reply:
@travis79 Just to double check, I searched Glean Dictionary and the only metrics from which we duplicated seems to be in the Glean repo [here](https://github.com/mozilla/glean/blob/015f77a7883b253a501fe3f17ddc613943c28276/glean-core/metrics.yaml#L20). Are these to be linked?

Reply:
Yes, that is correct. Please include the data-review link(s) from the Glean definitions in the definitions you are adding. That way the data-review that covers the collection of these is linked in the metadata and easy to find from the Glean Dictionary and other tools. You can also copy over the linked bugs, but that isn't as necessary as the data-reviews. Thanks!